examples/doc-scan/README.md

```diff
@@ -6,3 +6,6 @@
 1. Install the dependencies with `npm install`
 1. Start the server `npm start`
 1. Visit `https://localhost:3000`
+
+> * The [default controller](./src/controllers/index.controller.js) demonstrates how to create a Doc Scan session
+> * The [success controller](./src/controllers/success.controller.js) demonstrates how to retrieve a Doc Scan session
```

Comment:
Did you mean to put this in a quote? Looks a little strange to me. Maybe we could just do italics?
```suggestion
* _The [default controller](./src/controllers/index.controller.js) demonstrates how to create a Doc Scan session_
* _The [success controller](./src/controllers/success.controller.js) demonstrates how to retrieve a Doc Scan session_
```

Reply:
I normally put notes/info in a block like this - I can see why using a quote might be confusing for this